scrapyd/scripts/scrapyd_run.py

```diff
@@ -4,11 +4,21 @@
 from os.path import join, dirname
 from sys import argv
 import scrapyd
+import pkgutil
+
+
+def print_version():
+    __version__ = pkgutil.get_data(__package__, '../VERSION').decode('ascii').strip()
+    print(f'Scrapyd {__version__}')
 
 
 def main():
+    if len(argv) > 1 and argv[-1] in ['-v', '--version']:
```

Comment:
Hmm, should we just check whether the options appear in any of the argv arguments, e.g.

```suggestion
    if len(argv) > 1 and set(argv[1:]) & set(['-v', '--version']):
```

Reply:
Yes it looks reasonable